src/test/java/com/artipie/vertx/VertxSliceServerTest.java

```diff
@@ -139,6 +139,26 @@ public void basicGetRequest() {
         MatcherAssert.assertThat(actual, Matchers.equalTo(expected));
     }
 
+    @Test
+    public void basicGetRequestCL() {
```

Comment:
@Sammers21 please improve test name, it should describe that we test delivery of response with `Content-Length` header. It is not clear what `CL` here means